packages/suite/src/types/suite/guide.tsx

```diff
@@ -54,3 +54,25 @@ export type FeedbackSuggestion = {
 };
 
 export type Feedback = FeedbackBug | FeedbackSuggestion;
+
+export enum GuideType {
+    ARTICLE = 'page',
+    CATEGORY = 'category',
+}
+
+type GuideItemCommonType = {
+    id: string;
+    locales: string[];
+    title: { [key: string]: string };
+};
+
+export type GuideArticleType = {
+    type: GuideType.ARTICLE;
+} & GuideItemCommonType;
+
+export type GuideCategoryType = {
+    type: GuideType.CATEGORY;
+    children: GuideCategoryType[] | GuideArticleType[];
```

Comment:
Is it correct, that it should be array of items of one type only – category OR article? 

Maybe  `children: GuideItemType[]` would be more accurate?

Reply:
Resolved.